i3lock-fancy-rapid.c

```diff
@@ -93,12 +93,67 @@ void box_blur(unsigned char *dest, unsigned char *src, int height, int width,
 
 }
 
+void pixelate(unsigned char *dest, unsigned char *src, int height,
+                   int width, int radius)
+{
```

Comment:
Maybe
```suggestion
{
    radius = radius * 2 + 1;
```
according to the code below?
Because this is the definition of radius.